src/numberReducer.test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const numberReducer = require('./numberReducer');
+
+// increase tests
+test('reducer should increase the number (0 => 1)', () => {
+  expect(numberReducer(0, { type: 'increase' }))
+    .toBe(1);
+});
+
+test('reducer should increase the number (10 => 11)', () => {
+  expect(numberReducer(10, { type: 'increase' }))
+    .toBe(11);
+});
+
+test('reducer should increase the number (-1 => 0)', () => {
+  expect(numberReducer(-1, { type: 'increase' }))
+    .toBe(0);
+});
+
+// decrease tests
+test('reducer should decrease the number (0 => -1)', () => {
+  expect(numberReducer(0, { type: 'decrease' }))
+    .toBe(-1);
+});
+
+test('reducer should decrease the number (10 => 9)', () => {
+  expect(numberReducer(10, { type: 'decrease' }))
+    .toBe(9);
+});
+
+test('reducer should decrease the number (-1 => -2)', () => {
+  expect(numberReducer(-1, { type: 'decrease' }))
+    .toBe(-2);
+});
```

Comment:
It's enough to have only 1 test per each action
and 2 tests for `clear` 
numberReducer (10, { type: 'clear' }) === 0
numberReducer(0, { type: 'clear' }) === 0

And  1 unknown action test
numberReducer (10, { type: 'unknown' }) === 10